Sources/ProjectDescription/Cache.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+// TODO: Add documentation
```

Comment:
😬 We should address those before merging the PR.

Reply:
Absolutely, I am going to tackle all TODOs I added and fill in the documentation.